src/components/carousel/carousel.ts

```diff
@@ -34,10 +37,14 @@ export class MCarousel extends Vue {
     }
 
     protected updated(): void {
+        this.removeKeyboardListener();
```

Comment:
est-ce que c'est juste pour trapper le cas où a prop de navigation clavier est settée? Si oui, est-ce que ça ne serait pas plus simple de ne rien mettre dans le mounted et l'update, et de juste mettre un watch sur la prop?

Reply:
Un watch pourrait faire la même chose, mais avec un coût de performance plus élevé. Est-ce que c'est ce qu'on veut?

Reply:
Faudrait regarder de quelle manière c'est plus coûteux. Présentement, pour chaque appel à "update" on enlève et rajoute de listener entre autres.

Reply:
Effectivement, j'aurais dû mettre un if pour vérifier si si la prop a changé. Par contre, le watch va s'exécuter à chaque lifecycle de l'application, tandis que le updated s'exécute seulement si une props change sur l'instance de carousel.

Reply:
Je suis quand même curieux pour la question de performance. Selon ce que j'avais vu : "Keep in mind watchers only change when that specific data property changes". Peut-être qu'à l'interne de Vue, c'est fait sur chaque cycle... mais je ne vois pas pourquoi ce serait différent pour l'update à l'interne.

Reply:
Je pensais que les watch se faisaient à chaque lifecycle... alors se serait mieux d'utiliser un watch.